docs/datadoc/keywords.md

```diff
@@ -56,8 +56,8 @@ Resource published or curated by an agent.
 | [version]               | [rdfs:Literal]                                 | optional    | Version indicator (name or identifier) of a resource.                                                                                        |                                                                                                                                                                                                       |
 | [versionNotes]          | [rdfs:Literal]                                 | optional    | A description of the differences between this version and a previous version of the resource.                                                | This property can be repeated for parallel language versions of the version notes.                                                                                                                    |
-| [hasCurationDate]       | [rdfs:Literal]<br>(xsd:date)                   |             | The date the resource was curated.                                                                                                           | Should be used with `hasCurator`.                                                                                                                                                                     |
-| [hasCurator]            | [foaf:Agent]                                   |             | The agent that curated the resource.                                                                                                         |                                                                                                                                                                                                       |
+| [curatedBy]             | [foaf:Agent]                                   |             | The agent that curated the resource.                                                                                                         |                                                                                                                                                                                                       |
+| [curationDate]          | [rdfs:Literal]<br>(xsd:date)                   |             | The date the resource was curated.                                                                                                           | Should be used with `curatedBy`.                                                                                                                                                                      |
```

Comment:
I wonder if the date will be too restrictive. Curation typically wil take place over more than one day.  Or are we talking aobut the resource was documented.